csv2docx/cli.py

```diff
@@ -15,9 +30,13 @@
 @click.option(
     '--delimiter', '-d',
     default=";",
-    help='delimiter used in your csv. Default is \';\'')
-def convert(data, template, delimiter):
-    print ("Getting .docx template and .csv data files ...")
+    help='Delimiter used in your csv. Default is \';\'')
+@click.option(
+    '--path', '-p',
```

Comment:
Maybe destination is a better name?

Reply:
I would go with destination yes

Reply:
@davidverweij -- one minor issue with destination is that the `data` option has the `-d` flag ...

Reply:
i would go with -o output it feels standard